app/views/home/index.en.html.erb

```diff
@@ -8,20 +8,41 @@
       <%= service_name %>
     </h1>
 
-    <p class="govuk-body">
+    <% if FeatureFlags.means_journey.enabled? %>
+      <p class="govuk-body">
+      Use this service to apply for criminal legal aid in ongoing cases. The application will be means tested.
+      </p>
+      <p class="govuk-body">
+      Your client has a partner.
```

Comment:
This should say your client cannot have a partner 🙂 : https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=7670-22054&mode=design&t=SuTbkhVAFzZlXix0-0

Reply:
Thanks will amend! :)